integrations/docker/images/stage-2/chip-build-nuttx/setup.sh

```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```

Comment:
why the empty script?

Reply:
@andy31415 There is no additional configuration required, and I am afraid that missing this file will cause compilation errors, so I have added an empty file here.

Reply:
No other intergration docker files seem to need it, so it should not be required here. It may have been a requirement if using nuttx, however if we do not it should be fine to not have such a file.